app/src/main/java/com/spartahack/spartahack17/Presenter/PrizePresenter.java

```diff
@@ -0,0 +1,56 @@
+package com.spartahack.spartahack17.Presenter;
+
+import android.util.Log;
+
+import com.spartahack.spartahack17.Adapters.PrizeAdapter;
+import com.spartahack.spartahack17.Model.Prize;
+import com.spartahack.spartahack17.Retrofit.GSONMock;
+import com.spartahack.spartahack17.Retrofit.ParseAPIService;
+import com.spartahack.spartahack17.View.PrizeView;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import rx.Observable;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+/**
+ * Created by memuyskens on 10/5/16.
+ * SpartaHack-Android
+ */
+public class PrizePresenter extends RxPresenter<PrizeView, GSONMock.Prizes> {
+
+    private static final String TAG = "PrizePresenter";
+
+    public void updatePrizes() {
+        if (isViewAttached()) {
+            getView().showLoading();
+        }
+
+        Observable observable = ParseAPIService.INSTANCE.getRestAdapter().getPrizes()
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribeOn(Schedulers.io());
+
+        subscribe(observable);
+    }
+
+    @Override void onError(Throwable e) {
+        Log.e(TAG, e.toString());
```

Comment:
Well, the error string does trickle down to the view so I could understand going that route. Of course we don't have any error handling in the views yet. Is it typical to catch the errors and keep the app alive or just let it crash?